packages/utils/src/SDKErrors.ts

```diff
@@ -174,9 +174,21 @@ export class IdentityMismatchError extends SDKError {
   }
 }
 
-export class WsAddressNotSetError extends SDKError {
-  constructor() {
-    super('Node address to connect to not configured!')
+export class FullNodeConnectionNotSetError extends SDKError {
+  constructor(options?: ErrorOptions) {
+    super(
+      'A function tried to access the ApiPromise instance before it was set. Did you forget to call `Kilt.init({ api })`?',
```

Comment:
I prefer messages to be to the point, what do you think?
```suggestion
      'The blockchain API is not set. Did you forget to call `Kilt.connect(…)` or `Kilt.init(…)`?',
```